apps/consent/app/login/verification/form.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import React from "react";
+// @ts-ignore
+import { experimental_useFormState as useFormState } from "react-dom";
+import { submitFormTotp } from "./server-actions";
+import { submitForm } from "./server-actions";
+import TwoFaVerificationForm from "../../components/varification-components/two-fa-verification-code-form";
+import VerificationCodeForm from "../../components/varification-components/verification-code-form";
+import { toast } from "react-toastify";
+
+interface VerificationFormProps {
+  login_challenge: string;
+  loginId: string;
+  remember: string;
+  loginType: string;
+  value: string;
+}
+
+const VerificationForm: React.FC<VerificationFormProps> = ({
```

Comment:
`VerificationForm` naming is too broad. it's unclear from the name if this is a Totp or if this is the captcha or something else. we should use more precise naming so that there is more understand of what this is supposed to represented just by looking at the filename or function name.